### PR TITLE
spellcheck

### PR DESCRIPTION
**Pull request by mgheorghe**
ran spellchecker on the md files

### CODE REVIEW

File: dash-pipeline/dockerfiles/dev-notes/design-slimmer-bmv2.md [hunk 1]
```diff
@@ -35,10 +35,12 @@ run-switch:
 chris@chris-z4:~/chris-DASH/DASH/dash-pipeline$ make run-all-tests
```
Comment (chrispsommers) [inline dash-pipeline/dockerfiles/dev-notes/design-slimmer-bmv2.md:35]: Do me a favor and delete this dev-notes directory, it's obsolete.

File: documentation/high-avail/design/xsight-labs-ha-proposal-v1.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ The data channel is used to exchange the flow update messages.
 
 The channels are shown in this functional model of the HA synchronization mechanisms:
 
-![ha-functional-dagram.svg](images/ha-functional-diag.svg)
+![ha-functional-diagram.svg](images/ha-functional-diag.svg)
```
Comment (chrispsommers) [inline documentation/high-avail/design/xsight-labs-ha-proposal-v1.md:30]: Will break filename link, rename the file to match or leave alone.

**Review by chrispsommers — CHANGES_REQUESTED** [review]
Great job, you fixed hundreds of typos, mostly mine! Please see a couple of requested changes, thanks.

File: dash-pipeline/dockerfiles/dev-notes/design-slimmer-bmv2.md (hunk 1 above)
Comment (mgheorghe) [reply dash-pipeline/dockerfiles/dev-notes/design-slimmer-bmv2.md]: done

**Review by chrispsommers — APPROVED** [review]

**Comment by chrispsommers** [comment]
Hi @KrisNey-MSFT, Mircea did a pretty thorough job fixing hundreds of typos. Once this is accepted, he has another PR to do spell-checking in CI. Someone besides me should probably approve this, but I'm satisfied.

**Comment by KrisNey-MSFT** [comment]
LGTM

**Comment by KrisNey-MSFT** [comment]
Done!

**Pull request merged**